gradle-plugin/it/src/it/probes-groovy-dsl-config/expected/none/kubernetes.yml

```diff
@@ -0,0 +1,106 @@
+---
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      prometheus.io/path: /metrics
+      jkube.io/git-branch: "@ignore@"
+      prometheus.io/port: "9779"
+      prometheus.io/scrape: "true"
+      jkube.io/git-url: "@ignore@"
+      jkube.io/git-commit: "@ignore@"
+    labels:
+      expose: "true"
+      app: probes-groovy-dsl-config
+      provider: jkube
+      version: "@ignore@"
+      group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+    name: probes-groovy-dsl-config
+  spec:
+    ports:
+    - name: http
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      app: probes-groovy-dsl-config
+      provider: jkube
+      group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    annotations:
+      jkube.io/git-url: "@ignore@"
+      jkube.io/git-commit: "@ignore@"
+      jkube.io/git-branch: "@ignore@"
+    labels:
+      app: probes-groovy-dsl-config
+      provider: jkube
+      version: "@ignore@"
+      group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+    name: probes-groovy-dsl-config
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    selector:
+      matchLabels:
+        app: probes-groovy-dsl-config
+        provider: jkube
+        group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+    template:
+      metadata:
+        annotations:
+          jkube.io/git-url: "@ignore@"
+          jkube.io/git-commit: "@ignore@"
+          jkube.io/git-branch: "@ignore@"
+        labels:
+          app: probes-groovy-dsl-config
+          provider: jkube
+          version: "@ignore@"
+          group: org.eclipse.jkube.it.gradle.probes.groovy.dsl
+        name: probes-groovy-dsl-config
+      spec:
+        containers:
+        - env:
+          - name: KUBERNETES_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: HOSTNAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          image: dsl/probes-groovy-dsl-config:latest
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /actuator/health
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 180
+            successThreshold: 1
```

Comment:
I understand that this probe (and also the readiness one) comes from the Spring Boot Actuator detected dependency.

If this is the case, this test is not revealing its full potential, since the automatic probe and the configured probe are similar.

We should see if we can remove the generated probe (remove dependency), which would leave this manifest with no probes, or make the configured probe really different.

Reply:
Also we should consider the test-naming semantics depending on the above decision